modules/oadp-operator-supported.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-intro.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-operator-supported_{context}"]
+= {oadp-full} support
+
+.Supported versions of {oadp-short}
```

Comment:
Alternatively, you could modify the table heading and drop all of the previous, unsupported versions because that can be considered fluff. If a version is not found in the table of the currently supported versions, then it is simply not supported, and the details about its past support dates in history are no longer of any use or value. IOW, why should we keep on publishing information that is of no use to our customers/readers?
```suggestion
.Currently supported versions of {oadp-short}
```

Reply:
Things are either `Supported` or `Not supported`... `Currently` is not needed as it is redundant